tee-worker/build.Dockerfile

```diff
@@ -68,6 +68,7 @@ RUN \
   fi
 
 RUN cargo test --release --features development
+RUN cargo test -p lc-vc-task-receiver -- --ignored --test-threads 1 
```

Comment:
If there is no rush i suggest debugging `--test-threads` before this PR gets merged.

Reply:
The reason why I'm choosing to debug this in another issue, is because the most likely reason why it fails is because of the two globals we are using i.e. `GLOBAL_MOCK_RPC_API` and `GLOBAL_MOCK_AUTHOR_API`, it is something being used for only the unit testing, So I prefer doing it another PR. I would have to use a completely different mechanism to capture the values, and I think I should take a dig at it in a different/follow-up PR.